analytics/scala/src/test/resources/client.properties

```diff
@@ -1,5 +1,5 @@
 # Please register your application with your copy of Koverse before running
 # the examples in this package. To register your application follow the instructions at TODO
-client.name=examples
-client.secret=changeme
+client.name=testClient
+client.secret=c4243d17-e320-470d-b534-f9ea238e8adb
```

Comment:
I dont' think we should use the test client here. That's our secret. This repo is for customers. They should add their own client name and secret here

Reply:
absolutely. I forgot to change it back.